articles/precarious-data.md

```diff
@@ -1,51 +1,217 @@
+	- [How to Do It: Write Content onto IPFS and Publish the Hashes](#how-to-do-it-write-content-onto-ipfs-and-publish-the-hashes)
+		- [Do I have to worry about Bad Content coming onto my machine?](#do-i-have-to-worry-about-bad-content-coming-onto-my-machine)
+		- [Writing Content onto IPFS](#writing-content-onto-ipfs)
+		- [Pinning](#pinning)
+		- [Publishing the Hashes](#publishing-the-hashes)
+	- [Covering Your Bases: Strategies for Making your Content Resilient](#covering-your-bases-strategies-for-making-your-content-resilient)
+		- [Talk to a Librarian](#talk-to-a-librarian)
+		- [Achieving Redundancy](#achieving-redundancy)
+		- [Ensuring Availability](#ensuring-availability)
+		- [Ensuring Authenticity](#ensuring-authenticity)
+		- [Dealing with Versioning](#dealing-with-versioning)
+		- [Preserving Data](#preserving-data)
+- [Why the Established Tools Aren't Good Enough](#why-the-established-tools-arent-good-enough)
+	- [What's wrong with just moving the data to a new, trusted location?](#whats-wrong-with-just-moving-the-data-to-a-new-trusted-location)
+	- [Why isn't it enough to have everyone download copies of the data?](#why-isnt-it-enough-to-have-everyone-download-copies-of-the-data)
+	- [Can't we use the cloud to back up the data?](#cant-we-use-the-cloud-to-back-up-the-data)
+	- [Can Libraries Save the Day?](#can-libraries-save-the-day)
+
+<!-- /TOC -->
+
+# Instructions for Saving Endangered Data: It's time to get decentralized.
+
+## The Situation: Endangered Data Woven into a Precarious Web
 
 **precarious**
 adj. Dangerously lacking in security or stability: _a precarious posture; precarious footing on the ladder._
 adj. Subject to chance or unknown conditions.
 
-It happened with GeoCities. It's happening with climate data. It also happens in little ways every time someone takes a snapshot of a controversial tweet or facebook thread before it gets deleted. Our digital lives are woven into the World Wide Web and that web sits on a precarious foundation. We've been shoring up that foundation with abortive attempts to grab snapshots of content before it changes or disappears. It's time to migrate the web onto resilient, decentralized foundations, starting with the endangered data.
+We keep grabbing pieces of the web before they fall away.  It happened with GeoCities. It's happening with climate data. It also happens in little ways every time someone takes a snapshot of a controversial tweet or Facebook thread before it gets deleted. Our digital lives are woven into the World Wide Web and that web is woven from unstable fibers. We've resisted the degradation of that web with abortive attempts to grab snapshots of content before it changes or disappears, but this isn't enough. It's time to weave a more resilient web by using decentralized techniques. We can start by using these techniques to catch and hold endangered data.
```

Comment:
I think that you're mixing together the metaphor of the web as a web, and the concept of the Web as a thing. I'm not sure it works well. "We keep grabbing pieces of the web before they fall away" doesn't sound great to me, and I misparsed it the first couple of times I read it. Starting with a description of the web as data being hosted by providers who occasionally stop serving data might be good. As well, I don't know who "we" are - I thought you meant users, but users aren't saving the web. I'm not sure what your audience is for reading this, and you could frame that. 

The "little ways" comment misses that taking snapshots of Twitter was a major news story last year; there was a site that did this, and Twitter went after them - IIRC - for violating terms of service. I wouldn't call that little. I would say it is a way of saving data. 

"We can start by using these techniques to catch and hold endangered data." This introduces a third concept for a web - a web which we are using to catch and hold data. I think that three different meanings of web is a bit too complex for an introductory paragraph.